libr/core/visual.c

```diff
@@ -2054,27 +2072,45 @@ R_API int r_core_visual_cmd(RCore *core, const char *arg) {
 					core->seltab = 0;
 				}
 			} else {
-				core->curtab = 0;
-				if (core->printidx == R_CORE_VISUAL_MODE_PDDBG) {
-					core->print->cur = 0;
-					core->seltab++;
-					if (core->seltab > 2) {
-						core->seltab = 0;
-					}
-				} else {
-					core->seltab = 0;
-					ut64 f = r_config_get_i (core->config, "diff.from");
-					ut64 t = r_config_get_i (core->config, "diff.to");
-					if (f == t && f == 0) {
-						core->print->col = core->print->col == 1? 2: 1;
+				if (core->print->cur_enabled) {
+					core->curtab = 0;
+					if (core->printidx == R_CORE_VISUAL_MODE_PDDBG) {
+						core->print->cur = 0;
+						core->seltab++;
+						if (core->seltab > 2) {
+							core->seltab = 0;
+						}
 					} else {
+						core->seltab = 0;
+						ut64 f = r_config_get_i (core->config, "diff.from");
+						ut64 t = r_config_get_i (core->config, "diff.to");
+						if (f == t && f == 0) {
+							core->print->col = core->print->col == 1? 2: 1;
+						} else {
 #if 0
-// XXX WTF
-						ut64 delta = offset - f;
-						r_core_seek (core, t + delta, 1);
-						r_config_set_i (core->config, "diff.from", t);
-						r_config_set_i (core->config, "diff.to", f);
+							// XXX WTF
```

Comment:
Maybe kill this?

Reply:
i will clean code in a different PR, because this visual.c have lots of things to improve and im focusing on usability right  now